graphql_compiler/schema.py

```diff
@@ -281,6 +281,13 @@ def _parse_datetime_value(value):
     parse_literal=_unused_function,  # We don't yet support parsing Decimal objects in literals.
 )
 
+CUSTOM_SCALAR_TYPES = (
+    GraphQLDecimal,
+    GraphQLString,
```

Comment:
`String` as a custom type? Could this be a typo?

Reply:
Yes it is.